src/views/controller/UnitsView.vue

```diff
@@ -137,27 +137,25 @@ function getBulkActionsKebabMenuItems() {
           {{ t('standalone.dashboard.data_updated_every_seconds', { seconds: 10 }) }}
         </div>
       </div>
-      <div class="mb-8 flex flex-col items-start justify-between gap-6 xl:flex-row">
-        <div class="max-w-2xl text-gray-500 dark:text-gray-400">
-          {{ t('controller.units.page_description') }}
-        </div>
-        <div v-if="!isShownUnitsSkeleton" class="flex shrink-0 flex-row-reverse gap-6 xl:flex-row">
-          <NeButton kind="tertiary" size="lg" @click="isShownAddUnitModal = true" class="shrink-0">
-            <template #prefix>
-              <FontAwesomeIcon :icon="['fas', 'circle-plus']" aria-hidden="true" />
-            </template>
-            {{ t('controller.units.add_unit') }}
-          </NeButton>
-          <NeButton kind="secondary" size="lg" @click="forceReloadUnitsData" class="shrink-0">
-            <template #prefix>
-              <FontAwesomeIcon :icon="['fas', 'arrows-rotate']" aria-hidden="true" />
-            </template>
-            {{ t('controller.units.force_data_reload') }}
-          </NeButton>
-        </div>
+      <div class="mb-8 max-w-2xl text-gray-500 dark:text-gray-400">
+        {{ t('controller.units.page_description') }}
       </div>
+      <!-- <div v-if="!isShownUnitsSkeleton" class="flex shrink-0 flex-row-reverse gap-6 xl:flex-row"> //// 
+        <NeButton kind="tertiary" size="lg" @click="isShownAddUnitModal = true" class="shrink-0">
+          <template #prefix>
+            <FontAwesomeIcon :icon="['fas', 'circle-plus']" aria-hidden="true" />
+          </template>
+          {{ t('controller.units.add_unit') }}
+        </NeButton>
+        <NeButton kind="secondary" size="lg" @click="forceReloadUnitsData" class="shrink-0">
+          <template #prefix>
+            <FontAwesomeIcon :icon="['fas', 'arrows-rotate']" aria-hidden="true" />
+          </template>
+          {{ t('controller.units.force_data_reload') }}
+        </NeButton>
+      </div> -->
```

Comment:
Any reason for not deleting this?

Reply:
Yes, the reason is: oversight :smile_cat: